choco/virtctl/virtctl.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>virtctl</id>
+    <version>1.0.0</version>
+    <title>Virtctl</title>
+    <authors>Muhammad Rafay</authors>
```

Comment:
Are you going to maintain the package further? Providing updates for the next releases?